routes/user.js

```diff
@@ -43,30 +44,34 @@ router.post("/add", (req, res) => {
 router.get("/verify", (req, res) => {
   const email = gUnverifiedEmails.get(req.query.state);
```

Comment:
Should the `gUnverifiedEmails` Map be stored in Postgres as well?

```js
const gUnverifiedEmails = new Map();
```

Reply:
Filed #59